app/src/hooks/api/useObservationApi.ts

```diff
@@ -21,9 +26,27 @@ const useObservationApi = (axios: AxiosInstance) => {
 
     return data;
   };
+  /**
+   * Create a new block observation
+   *
+   * @param {ICreateBlockObservationRequest} blockObservation
+   * @return {*}  {Promise<ICreateBlockObservationResponse>}
+   */
+  const createBlockObservation = async (
```

Comment:
this function is not named correctly
it is now a generic endpoint so should be called `createObservation` as it deals with not only block type anymore :)